api/rds.proto

```diff
@@ -20,15 +20,12 @@ import "validate/validate.proto";
 // configurations. Each listener will bind its HTTP connection manager filter to
 // a route table via this identifier.
 service RouteDiscoveryService {
-  rpc StreamRoutes(stream DiscoveryRequest)
-      returns (stream DiscoveryResponse) {
-  }
+  rpc StreamRoutes(stream DiscoveryRequest) returns (stream DiscoveryResponse) {}
 
-  rpc FetchRoutes(DiscoveryRequest)
-      returns (DiscoveryResponse) {
+  rpc FetchRoutes(DiscoveryRequest) returns (DiscoveryResponse) {
     option (google.api.http) = {
-      post: "/v2/discovery:routes"
-      body: "*"
+      post : "/v2/discovery:routes"
```

Comment:
Also weird spaces. Is there some standard clang-format config we should adopt for proto?